source/components/organisms/CheckboxList/CheckboxList.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState, useEffect } from "react";
+import styled from "styled-components/native";
+
+import { Checkbox, Text } from "app/components/atoms";
+import { ComplementaryColor } from "../../../styles/themeHelpers";
+
+interface CheckboxListProps {
+  choices: { displayText: string; tags?: string[]; id: string }[];
+  onChange?: (values: Record<string, boolean>) => void;
+  onBlur?: (values: Record<string, boolean>) => void;
+  value?: Record<string, boolean>;
+  colorSchema?: ComplementaryColor;
+}
+
+const BoxTextWrapper = styled.View`
```

Comment:
@karlknowit har recently been testing moving everything style related into a separate file which I think is a good structure. Would this be applicable here? That would mean moving `BoxTextWrapper`, `sizes`, and `CheckboxFieldText` into a separate `CheckboxList.styled.ts` file next to this file.